package.json

```
@@ -6,6 +6,9 @@
     "prettierTarget": "**/*.{js,ts,md,mdx,yml}"
   },
   "scripts": {
+    "test": "playwright test",
+    "test:debug": "playwright test --debug",
+    "test:unit": "jest tests/jest --maxConcurrency=8 --bail 1",
```

This is technically correct, but the convention is that `test` runs all tests, and `test:xxx` runs a particular subtest. 

```suggestion
    "test:e2e": "playwright test",
    "test:debug": "playwright test --debug",
    "test:unit": "jest tests/jest --maxConcurrency=8 --bail 1",
```